Packs/Base/ReleaseNotes/1_11_9.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### FindSimilarIncidentsByText
+Fixed query in get_incidents_by_time
```

Comment:
please write something that starts with "Fixed an issue where..."
Think of this from the customer's perspective what did he see that did not work and now is working.